runtime-testsuite/resources/org/antlr/v4/test/runtime/descriptors/LexerExec/UnicodeCharSet.txt

```diff
@@ -6,7 +6,7 @@ Lexer
 
 [grammar]
 lexer grammar L;
-ID : ([A-Z_]|'Ā'..'￾') ([A-Z_0-9]|'Ā'..'￾')*;
+ID : ([A-Z_]|'Ā'..'\uFFFC') ([A-Z_0-9]|'Ā'..'\uFFFC')*; // FFFD+ are not valid char
```

Comment:
\FFFD+ aren't valid char though. I suppose someone could be parsing 16 bit unsigned char?

Reply:
the other problem is we cannot allow `\uFFFF` as that is still special since the high bit is set. it means that the next 16bit word contains the remainder of the 32bit value. I think I'm ok requiring that binary parses in bytes and characters use valid characters.